packages/libs/eda/src/lib/core/components/computations/plugins/abundance.tsx

```diff
@@ -87,8 +87,8 @@ export const plugin: ComputationPlugin = {
             <>
               <br />
               <span>
-                Ranked abundance: Variables with {config.rankingMethod} = 0
-                removed. Showing up to the top eight variables.
+                Ranked abundance: Overlay variables with {config.rankingMethod}{' '}
```

Comment:
Is it possible to make the two subtitles more similar by having this part be `...with {config.rankingMethod} = 0 removed...`? Or alternatively can we change the subtitle at lines 61/62 to be `with {config.rankingMethod}{' '}= 0 removed`?

Reply:
@asizemore I found it interesting. I have tested and realized that actually that part, {' '}, is automatically added by pre-commit tool(s) (probably prettier). So I cannot change or add that part by myself: it is treated as empty commit. 

Thus we just need to leave it as is

Reply:
wild. Good to know!